addons/l10n_es_pos/models/account_move.py

```diff
@@ -10,3 +10,8 @@ def _compute_l10n_es_is_simplified(self):
         for move in self:
             if move.pos_order_ids:
                 move.l10n_es_is_simplified = move.pos_order_ids[0].is_l10n_es_simplified_invoice
+
+    def _generate_pdf_and_send_invoice(self, template, force_synchronous=True, allow_fallback_pdf=True, bypass_download=False, **kwargs):
+        if self.company_id.country_code == "ES" and not self.company_id.l10n_es_edi_facturae_certificate_id:
```

Comment:
Hi @robinengels @jbw-odoo, this is causing some issues for database where the `l10n_es_edi_facturae` module is not installed, cf <https://github.com/odoo/odoo/pull/181346>.

I don't see anything specific to the PoS here, so I'm wondering if that changeset could be moved to within `l10n_es_edi_facturae` (instead of `l10n_es_pos`). What do you think ?

Reply:
Indeed, there is nothing related to the PoS in the code. However the issue concerns only the PoS so we might add a condition to check that the `account.move` is linked to a pos_order

Reply:
Ok I'll move the code to the other module. I'll link the PR here and close the fw port in 18 and master